src/SQLite3-Core-Tests/ManifestSQLite3CoreTests.class.st

```diff
@@ -0,0 +1,23 @@
+"
+Please describe the package using the class comment of the included manifest class. The manifest class also includes other additional metadata for the package. These meta data are used by other tools such as the SmalllintManifestChecker and the critics Browser
+"
+Class {
+	#name : #ManifestSQLite3CoreTests,
+	#superclass : #PackageManifest,
+	#category : #'SQLite3-Core-Tests-Manifest'
+}
+
+{ #category : #'code-critics' }
+ManifestSQLite3CoreTests class >> ruleEmptyExceptionHandlerRuleV1FalsePositive [
+	^ #(#(#(#RGMethodDefinition #(#SQLite3ConnectionTest #tearDown #false)) #'2022-05-10T08:25:11.111+02:00') #(#(#RGMethodDefinition #(#SQLite3BaseConnectionTest #tearDown #false)) #'2022-05-10T08:27:13.419+02:00') )
+]
+
```

Comment:
We can ignore the ManifestXXX classes from the coverage reporting by implementing `classNamesNotUnderTest`. So it will not influence the coverage calculation.

Reply:
@gcotelli Where do we need to implement that? Do you have an example somewhere?

Reply:
AFAIR, it can be implemented as a class method in any of the package classes. So I suggest implementing it in `ManifestSQLite3CoreTests`

Reply:
Done in https://github.com/pharo-rdbms/Pharo-SQLite3/pull/49/commits/acfcce3f91aed7ba2e6ad88b8e24b5fd4faa8b21